aten/src/ATen/native/cpu/BinaryOpsKernel.cpp

```diff
@@ -673,6 +673,24 @@ void huber_kernel(TensorIterator& iter, double delta) {
 
 void sigmoid_backward_kernel(TensorIterator& iter) {
   AT_DISPATCH_FLOATING_TYPES_AND2(kBFloat16, kHalf, iter.dtype(), "sigmoid_backward_cpu", [&]() {
+    if (iter.dtype() == kBFloat16) {
```

Comment:
This looks weird, can you please do `if` statement otside of dispatch

Reply:
Yeah... good idea. Fixed!